app/controllers/home_controller.rb

```diff
@@ -1,7 +1,7 @@
 class HomeController < ApplicationController
   def index
     @interaction_types = ApplicationController.helpers.interaction_types
-    @environments = %w[gov.uk staging.publishing.service.gov.uk integration.publishing.service.gov.uk]
+    @environments = %w[integration.publishing.service.gov.uk staging.publishing.service.gov.uk gov.uk]
```

Comment:
I swapped these around, just so that integration is the default option on the form - so production isn't spammed with data